services/src/api/model/services.rs

```diff
@@ -99,3 +100,14 @@ pub enum DataPath {
     Volume(VolumeName),
     Upload(UploadId),
 }
+
+// ------------------------------------------------------------------------------------
+// ML Model
+// ------------------------------------------------------------------------------------
+
+#[derive(Debug, Serialize, Deserialize, Clone, ToSchema)]
+#[serde(rename_all = "camelCase")]
+pub struct AddMlModel {
```

Comment:
What is this for? It is not used anywhere yet.

Reply:
Old code that accedentially made it into the commit.

99642a72f6991cb2c4aff26f96dc15f6e025b300

--- 
Commit MSG:

chore: remove dead code (AddMlModel)